components/Search/Search.jsx

```diff
@@ -142,7 +143,7 @@ const Search = ({
           )
         )}
       </div>
-      {error && <ErrorMessage />}
+      {error && data != 1 && <ErrorMessage />}
```

Comment:
What happens if for some reason there is an error (say a HTTP error) and the value of `data` is `1`? Is that scenario possible?

Reply:
It should be not possible, i've  tested it with a "wrong" connection string and it returns an empty array - triggering the error correctly